webnn.json

```diff
@@ -242,6 +264,19 @@
       {"name": "activation", "type": "operator", "optional": true}
     ]
   },
+  "gru options": {
+    "category": "structure",
+    "members": [
+      {"name": "bias", "type": "operand"},
```

Comment:
Add `"optional": true` for bias, recurrent bias, initial hidden state.

Reply:
Optimized.